test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs

```diff
@@ -8,7 +8,7 @@
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
-[assembly: CollectionBehavior(MaxParallelThreads = 12)]
+[assembly: CollectionBehavior(MaxParallelThreads = 4)]
```

Comment:
Do we know how many it's using at the moment? i.e. how many virtual CPUs do we have?

Reply:
afaik, 2

Reply:
They run on Standard_DS2_v2: https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series

Reply:
Then I know how to speed up these tests 😉  (`Standard_DS3_v2`)

Reply:
I wish :D But we have no control over it, short of running our own instances: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#hardware